docs/google-sign-in.rst

```diff
@@ -0,0 +1,28 @@
+.. _google-sign-in:
+
+Google Sign-In
+==============
+
+Google Sign-In for the Assistant provides the simplest and easiest user experience to users and developers both for account linking and account creation. Your Action can request access to your user's Google profile during a conversation, including the user's name, email address, and profile picture.
+
+The profile information can be used to create a personalized user experience in your Action. If you have apps on other platforms and they use Google Sign-In, you can also find and link to an existing user's account, create a new account, and establish a direct channel of communication to the user.
+
+To perform account linking with Google Sign-In, you ask the user to give consent to access their Google profile. You then use the information in their profile, for example their email address, to identify the user in your system. Check out this `link <https://developers.google.com/actions/identity/google-sign-in>`_ for more information.
+
+With Voxa, you can ask for the user's full name like this:
+
+.. code-block:: javascript
+
+  app.onIntent('ProfileIntent', async (dialogFlowEvent: DialogFlowEvent) => {
+    const userInfo = await dialogFlowEvent.getUserInformation();
+
+    dialogFlowEvent.model.email = userInfo.email;
```

Comment:
is there any chance that is values are empty? if so do it should be specified in here.

Reply:
I'm not totally sure, I have made some tests and I do get values in those variables